templates/SilverStripe/Forms/CompositeField_holder.ss

```diff
@@ -1,4 +1,4 @@
-<$Tag class="CompositeField $extraClass <% if $ColumnCount %>multicolumn<% end_if %>" id="$HolderID">
+<$Tag class="CompositeField $extraClass <% if $ColumnCount %>multicolumn<% end_if %>" id="$HolderID" $AttributesHTML>
```

Comment:
Won't this re-render the class attribute? You might need to use `$getAttributesHTML('class')` to exclude it

Reply:
Agreed, there's more Attributes duplicated. Will make an other commit, plz close this one

Reply:
You can use the same branch, no need for another pull request =)

Reply:
Ah I see - just haven't tried before to do it at the github without downloading repo) Fixed the issues